mixed-persistence/mixed-persistence-java-sbt/hello-api/src/main/java/com/lightbend/lagom/samples/mixedpersistence/hello/api/GreetingMessage.java

```diff
@@ -3,7 +3,18 @@
 import lombok.NonNull;
 import lombok.Value;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
These are now required, I'm not sure if this is because Lombok or Jackson update (there was an update in Play to 2.9.9).